python/fledge/services/core/api/plugins/common.py

```diff
@@ -32,8 +32,9 @@
 
 def load_python_plugin(plugin_module_path: str, plugin: str, _type: str) -> Dict:
     _plugin = None
+    module_name = "fledge.plugins.{}.{}.{}".format(_type, plugin, plugin)
```

Comment:
Please remove .{} plugin. Having `module_name = "fledge.plugins.{}.{}".format(_type, plugin) is enough, so that we can avoid too much of plugin name in logs.

Reply:
sure